qa/L0_s3_local/test.sh

```diff
@@ -71,13 +79,13 @@ export AWS_ACCESS_KEY_ID=minio && \
 
 # create and add data to bucket
 python -m pip install awscli-local && \
-    awslocal --endpoint-url=http://localhost:4572 s3 mb s3://demo-bucket && \
-    awslocal s3 sync $DATADIR s3://demo-bucket
+    awslocal --endpoint-url=http://localhost:4572 mb s3://demo-bucket1.0 && \
```

Comment:
Need an underscore in the bucket name and an upper case.

Reply:
The bucket name can contain only lower-case characters, numbers, periods, and dashes